x/mongo/driver/description/server.go

```diff
@@ -239,13 +240,32 @@ func NewServer(addr address.Address, response bsoncore.Document) Server {
 				return desc
 			}
 			desc.SetVersion = uint32(i64)
+		case "speculativeAuthenticate":
```

Comment:
rebasing issue?

Reply:
whoever added it to operation/ismaster forgot to add it here

Reply:
Ah, gotcha

Reply:
I added support for `speculativeAuthenticate`. I changed the codepath that actually parses `isMaster` responses from the server in `operation/ismaster.go` but not the one here. Nothing failed because there aren't any SDAM spec tests that require `speculativeAuthenticate` support, so not a bug or anything, but glad we consolidated these paths now.